azure/multiapi/cosmosdb/v2017_04_17/common/_connection.py

```diff
@@ -0,0 +1,167 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# --------------------------------------------------------------------------
+import sys
+
+if sys.version_info >= (3,):
+    from urllib.parse import urlparse
+else:
+    from urlparse import urlparse
+
+from ._constants import (
+    SERVICE_HOST_BASE,
+    DEFAULT_PROTOCOL,
+    DEV_ACCOUNT_NAME,
+    DEV_ACCOUNT_KEY,
+    DEV_BLOB_HOST,
+    DEV_QUEUE_HOST,
+    DEV_TABLE_HOST
+)
+from ._error import (
+    _ERROR_STORAGE_MISSING_INFO,
+)
+
+_EMULATOR_ENDPOINTS = {
+    'blob': DEV_BLOB_HOST,
+    'queue': DEV_QUEUE_HOST,
+    'table': DEV_TABLE_HOST,
+    'file': '',
+}
+
+_CONNECTION_ENDPOINTS = {
+    'blob': 'BlobEndpoint',
+    'queue': 'QueueEndpoint',
+    'table': 'TableEndpoint',
+    'file': 'FileEndpoint',
+}
+
+_CONNECTION_ENDPOINTS_SECONDARY = {
+    'blob': 'BlobSecondaryEndpoint',
+    'queue': 'QueueSecondaryEndpoint',
+    'table': 'TableSecondaryEndpoint',
+    'file': 'FileSecondaryEndpoint',
+}
```

Comment:
Why does cosmosdb/table still include blob queue file endpoints?
Is this just how it's implemented in their SDK?